bonfire/bonfire.py

```diff
@@ -904,12 +904,23 @@ def _cmd_namespace_wait_on_resources(namespace, timeout, db_only):
 
 @namespace.command("describe")
 @click.argument("namespace", required=False, type=str)
-def _describe_namespace(namespace):
+@click.option(
+        "--output",
+        "-o",
+        default="cli",
+        help="which output format to return the data in",
+        type=click.Choice(["cli", "json"], case_sensitive=False),
+)
+def _describe_namespace(namespace, output):
     """Get current namespace info"""
     if not namespace:
         namespace = current_namespace_or_error()
-
-    click.echo(describe_namespace(namespace))
+    if output == "json":
+        data = {}
+        data = describe_namespace(namespace, output)
+        click.echo(json.dumps(data, indent=2))
```

Comment:
To simplify this, could you just have the `describe_namespace` function return `json.dumps(data, indent=2)` when `output == "json"` (so that func always returns a string) and just keep line 912 as it was previously?

Reply:
please check now @bsquizz